assets/css/app.css

```diff
@@ -455,3 +455,16 @@ select {
 .alert-info {
   background: #00C6B3;
 }
+
+/* Burger menu */
```

Comment:
consider adding the issue number to the comment for _even more_ clarity for "future self". 😉